css/order.css

```diff
@@ -398,3 +364,191 @@ svg {
   text-align: center;
   transition: 300ms all ease-in-out;
 }
+
+#bnomics-address-input,
+#bnomics-amount-input {
+  padding: 5px;
+  max-width: 190%;
```

Comment:
`max-width: 190%;` does not appear to be necessary. Also `max-width: 580px;` below.